src/Microsoft.VisualStudio.ProjectSystem.IntegrationTests/CSharp/CSharpProjectFileEditTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem.IntegrationTests
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpProjectFileEditTest : AbstractIntegrationTest
+    {
+        protected override string DefaultLanuageName => LanguageNames.CSharp;
```

Comment:
This appears to be an existing typo. I'll update all integration tests to use the correct name.